src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6343,12 +6398,191 @@
 
         public override IOperation? VisitInterpolatedStringHandlerCreation(IInterpolatedStringHandlerCreationOperation operation, int? captureIdForResult)
         {
-            return VisitNoneOperation(operation, captureIdForResult);
+            // We turn the interpolated string into a call to create the handler type, a series of append calls (potentially with branches, depending on the
```

Comment:
>Contains [](http://example.com/codeflow?start=95&length=8)

It looks like we are leaving the room for the ```Contains``` to not find the match. I.e. we will not assert here, at the same time, it looks like we will use _currentInterpolatedStringHandlerArgumentContext regardless.